Makefile

```diff
@@ -173,7 +173,7 @@ buildx:
 ###### Devkit container image
 DEVKIT_IMAGE_DOCKERFILE ?= Dockerfile.devkit
 DEVKIT_IMAGE_NAME ?= mesosphere/konvoy-image-builder-devkit
-DEVKIT_IMAGE_TAG ?= $(shell cat ${DEVKIT_IMAGE_DOCKERFILE} requirements.txt requirements-devkit.txt  | sha256sum | cut -d" " -f 1)
+DEVKIT_IMAGE_TAG ?= $(shell cat ${DEVKIT_IMAGE_DOCKERFILE} requirements.txt requirements-devkit.txt ansible/group_vars/all/defaults.yaml  | sha256sum | cut -d" " -f 1)
```

Comment:
nit: you may want to have a separate PR to make it clearer when backporting and PR against main

Reply:
💯

Reply:
https://github.com/mesosphere/konvoy-image-builder/pull/1196